### ISSUE TITLE
Check affinity in get_physical on Linux

### ISSUE THREAD

**Issue by seanmonstar** [seanmonstar/num_cpus#40]
The `get_physical` implementation for Linux should consider the affinity mask when collecting the number of physical cores on the system.

**Comment by cedenday** [comment]
Should this be closed? This is now in v1.5.0.

**Comment by seanmonstar** [comment]
No, it's only checked in `get()`, more work is needed to check the mask in `get_physical()`.
